radio/src/gui/colorlcd/view_text.cpp

```diff
@@ -466,7 +466,7 @@ void readModelNotes()
 #if !defined(EEPROM) 
   if(!notesFound) {   
     modelNotesName.assign(g_eeGeneral.currModelFilename);
-    size_t index = modelNotesName.find(".bin");
+    size_t index = modelNotesName.find(".yml");
```

Comment:
Probably better to use `YAML_EXT` here, and you could use a `#if defined(SDCARD_YAML)` to make it to use one or the other (YAML_EXT or MODELS_EXT)

Reply:
Thank you! I did not know about these defines.

Reply:
No problem, it's amazing what weird information you remember when poking around the code :grin: